clang/include/clang/Driver/Options.td

```diff
@@ -6919,6 +6919,10 @@ defm loop_versioning : BoolOptionWithoutMarshalling<"f", "version-loops-for-stri
 
 def fhermetic_module_files : Flag<["-"], "fhermetic-module-files">, Group<f_Group>,
   HelpText<"Emit hermetic module files (no nested USE association)">;
+
+def do_concurrent_to_openmp_EQ : Joined<["-"], "fdo-concurrent-to-openmp=">,
+  HelpText<"Try to map `do concurrent` loops to OpenMP [none|host|device]">,
```

Comment:
Is it `Try` because it will bail after an analysis?

Reply:
Yes, I think this will be a best-effort transformation, hopefully with good diagnostics on which loops whee transformed and which were not.